app/views/admin/booths/index.html.haml

```diff
@@ -9,6 +9,22 @@
             = link_to 'Add Booth', new_admin_conference_booth_path(@conference.short_title), class: 'button btn btn-primary'
       %p.text-muted
         All the booth requests
+
+      - if @conference.booth_limit == 0
+        %p
+          Set the
+          = link_to 'Booth limit', edit_admin_conference_path(@conference.short_title)
+          to make sure you are not accepting more booth requests than possible.
```

Comment:
what about this wording "to make sure you are not accepting more booths than you can accommodate" ?